### PR TITLE
Update authors.yml

### PR DESCRIPTION
**Pull request by hpcnotes**

### CODE REVIEW

**Comment by vsoch** [comment]
Taking a look!

**Comment by vsoch** [comment]
Heyo! So it looks like you cloned the wrong blog? https://github.com/hpcnotes/blog.hpc.social The clone should be for the personal blog here. Let me know if you want to redo the PR, or if you'd like me to add for you.

**Comment by hpcnotes** [comment]
Let me try again (in order to learn - I've actually only ever used github for reading stuff before, never contributing!)

**Comment by vsoch** [comment]
Awesome! Ok I’m here to help if you need.

**Comment by hpcnotes** [comment]
Ok, I looked again - the links in the instructions at the bottom of https://hpc.social/blog/ point you to fork the top level blog - presumably before it was split into personal and community blogs - so my first fork was to https://github.com/hpcnotes/blog.hpc.social as you noted - but once I figured out that doesn't actually have a _data/authors file, I then forked the personal blog https://github.com/hpc-social/personal-blog to https://github.com/hpcnotes/hpc-social-personal-blog ... and hopefully that is where I made my edit and pull request from?

**Comment by vsoch** [comment]
It's correct that the links on that page should direct to the main blogs page - that's the central place with instructions to not need to write them in multiple. What I'll do is add a clarifying point here:

image

that the clone is NOT for the portal repo there, but the blog they have chosen in the step above.

And yes - if you fork and PR from the personal-blog that is correct!

**Comment by vsoch** [comment]
OK warning / note added with an emoji emphasis! 
image

**Comment by hpcnotes** [comment]
OK, thanks, that makes more sense - big yellow stickers for people like me :-)

**Comment by vsoch** [comment]
> OK, thanks, that makes more sense - big yellow stickers for people like me :-)

And me too! :laughing:
